add_modules/dna_rna_tools_module.py

```diff
@@ -0,0 +1,51 @@
+nucl_dna = {"a", "t", "g", "c", "A", "T", "G", "C"}
+nucl_rna = {"a", "u", "g", "c", "A", "U", "G", "C"}
+    
+def is_na(seq: str) -> bool:
+    if set(seq) <= (nucl_dna) or set(seq) <= (nucl_rna):
+        return True
+    else:
+        return False
```

Comment:
Вот так мы прям на 3 лекции разбирали почему не надо делать!
```suggestion
def is_na(seq: str) -> bool:
    return set(seq) <= (nucl_dna) or set(seq) <= (nucl_rna):
```

Reply:
Не надо проверять логическое значение логического утверждения чтобы вернуть другое логическое значение. Можо сразу возвращать что есть :)